constants/stackNavigatorNames.js

```diff
@@ -14,4 +14,7 @@ export default {
   MULTI_SELECT: 'Multi Select',
   SELECT: 'Select',
   ANNOUNCEMENT_PREVIEW: 'Announcement Preview',
+  MY_ANNOUNCEMENTS: 'My Announcements',
+  MY_FOLLOWED_ANNOUNCEMENTS: 'My followed announcements',
```

Comment:
każdy wyraz powinien zaczynac się z dużych liter

Reply:
jeszcze tutaj dopisze, jak masz ulubione ogłoszenia to zmień tam title na brak obserwowanych ogłoszeń :D jak faktycznie ich nie masz